src/Shared/E2ETesting/E2ETesting.targets

```diff
@@ -114,4 +131,8 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_EnsureSeleniumScreenShotsFolder" BeforeTargets="Build">
```

Comment:
Why do this here and [here](https://github.com/aspnet/AspNetCore/pull/13672/files#diff-f24f8f4e4c7e3f2faa3a7f6b5eb77f82R95)? Seems like the C# ought to be sufficient.

Reply:
I want to avoid the runtime code for creating the folder when running on the CI